tests/test_board.py

```diff
@@ -1,70 +1,113 @@
 from engine.board import Board
 from engine.cardstack import CardStack
+
 import pytest
-import unittest
+import util
+
+
+@pytest.fixture(scope="module")
+def card():
+    return util.create_mock_card(number=2, color='blue')
+
+
+@pytest.fixture(scope="module")
+def your_card():
+    return util.create_mock_yourcard(public_number=False, public_color=False)
+
+
+@pytest.fixture(scope='module')
+def mock_deck():
+    return util.create_mock_deck(card_colors=('red', 'blue'), card_numbers=(1, 2, 2, 3), r_len=40)
+
+
+@pytest.fixture(scope='function')
+def board(mock_deck):
+    return Board(mock_deck)
+
+
+def test_init_creates_expected_cardstacks(board, mock_deck):
+    # Check that there's a stack for each color in the deck
+    for color in mock_deck.card_colors:
+        assert isinstance(board.card_stacks[color], CardStack)
+
+
+def xtest_init_creates_discard_stats(board, mock_deck):
+    # Assert there's a discard pile with a length:
+    assert len(board.discard) == 0
+    # Assert there's a discard_stats list with one counter per card number
+    for color in mock_deck.card_colors:
+        assert len(board.discard_stats[color]) == len(set(mock_deck.card_numbers))
+
+
+def test_discard_card_to_discard(board, card):
+    assert card not in board.discard
+    board.discard_card(card)
+    assert card in board.discard
+
+
+def test_discard_card_your_card(board, your_card):
```

Comment:
Yes, this will likely change in the future. I wasn't sure of the behavior and how it relates to the rest of the engine. This *tests* the currently implemented behavior.